ipalib/x509.py

```diff
@@ -285,7 +289,7 @@ def extended_key_usage_bytes(self):
             return None
 
         ekurfc = rfc2459.ExtKeyUsageSyntax()
-        for i, oid in enumerate(eku):
+        for i, oid in enumerate(sorted(eku)):
```

Comment:
I'm curious: what's the reason for this change?

Reply:
EKU is a set of OIDs. Without sort, the sorting of EKU items depends on Python's hash randomziation. The change makes the sort order of EKUs stable. It simplifies testing and makes the output predictable.